server/test/YmlKaoFileVisitor.test.ts

```diff
@@ -249,7 +253,7 @@ describe('Extension Server Tests', () => {
         });
         // tslint:disable-next-line: max-line-length
         it('should parse a well-written YML file containing instances and functions and provide completion for them', (done) => {
-            const inputStream = new ANTLRInputStream(`
+            const inputStream = CharStreams.fromString(`
```

Comment:
By the way, what triggered this change?

Reply:
The upgrade of the ANTLR dependency makes `ANTLRInputStream` deprecated.